samples/ios-cocoapods-static-framework/local-check.sh

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2024 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+#
+
+set -e
+
+log() {
+  echo "\033[0;32m> $1\033[0m"
+}
+
+if ! command -v xcodebuild &> /dev/null
+then
+    echo "xcodebuild could not be found, skip ios checks"
+    log "ios-cocoapods-static-framework not checked"
+
+    exit 0
+fi
+
+./gradlew clean compileKotlinIosX64
+log "ios-cocoapods-static-framework ios success"
+
+./gradlew clean podspec build generateDummyFramework
```

Comment:
here too we should not skip full build on nonmacos

Reply:
rework scripts